app/javascript/__tests__/modules/listingDetails/__snapshots__/ListingDetailsEligibility.test.tsx.snap

```diff
@@ -282,7 +282,7 @@ exports[`ListingDetailsEligibility displays listing details eligibility section
             </p>
           </div>
           <button
-            className="button-toggle ml-4"
```

Comment:
<img width="575" alt="Screen Shot 2022-11-14 at 3 49 18 PM" src="https://user-images.githubusercontent.com/65367387/201762689-46712454-734e-4813-9038-f48ce54a9b5c.png">
This change fixes this visual issue (screenshot is from full)